docs/user-guide.md

```diff
@@ -129,20 +156,20 @@ the values that are assigned to the variable. For example:
 
 ```java
 @NullMarked
-public class MyClass {
+class MyClass {
   void myMethod(@Nullable String one, String two) {
-    String copyOfOne = one;
-    String copyOfTwo = two;
+    String anotherOne = one;
+    String anotherTwo = two;
     String oneOrTwo = random() ? one : two;
     String twoOrNull = Strings.emptyToNull(two);
     ...
   }
 }
 ```
 
-Analysis can tell that all of these variables except `copyOfTwo` can be null.
-`copyOfTwo` can't be null since `two` can't be null: it is not `@Nullable` and it is
-inside the scope of `@NullMarked`. `copyOfOne` can be null since it is a copy of a
+Analysis can tell that all of these variables except `anotherTwo` can be null.
+`anotherTwo` can't be null since `two` can't be null: it is not `@Nullable` and it is
+inside the scope of `@NullMarked`. `anotherOne` can be null since it is a copy of a
```

Comment:
Instead of `copy of` I would say `alias of` or `reference to`, same in the next sentence.

Reply:
I changed it to "assigned from".